src/plugins/archive.js

```diff
@@ -1,18 +1,24 @@
-const { createError } = require('../lib/result');
-const { stringBuilder, success, failure } = require('../lib/format');
+const { createError, createWarning } = require('../lib/result');
+const { stringBuilder, success, failure, warning } = require('../lib/format');
 const { fetchGithub } = require('../lib/fetch');
 
 const archivePlugin = async (pkg, _, options) => {
+
+  const output = stringBuilder(
+    'Checking if github repository is archived'
+  ).withPadding(65);
+
+  if (!pkg.repository.url) {
```

Comment:
If the `repository` field is also undefined then the if statement will throw a TypeError.

```suggestion
  if (!pkg?.repository?.url) {
```